set.mm

```diff
@@ -440801,8 +440801,8 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
   latexdef "mmu" as "\mu";
 htmldef "sigma" as
     " <IMG SRC='sigma.gif' WIDTH=10 HEIGHT=19 ALT=' sigma' TITLE='sigma'> ";
-  althtmldef "sigma" as " &sigma; ";
-  latexdef "sigma" as " \sigma ";
+  althtmldef "sigma" as "&sigma;";
+  latexdef "sigma" as "\sigma";
```

Comment:
sigma is also an operation - therefore, there should be leading and trailing blanks (seee also html representation of "mod".